multichannel_timeseries/large_multichan.ipynb

```diff
@@ -0,0 +1,822 @@
+    "- **[Dask](https://github.com/dask/dask):** Adds parallel computing capabilities, managing tasks that exceed memory limits.\n",
+    "- **[ndpyramid](https://github.com/carbonplan/ndpyramid):** Specifically designed for creating multi-resolution data pyramids.\n",
+    "- **[Zarr](https://github.com/zarr-developers/zarr-python):** Used for storing the large arrays of the data pyramid on disk in a compressed, chunked, and memory-mappable format, which is crucial for efficient data retrieval.\n",
+    "- **[tsdownsample](https://github.com/predict-idlab/tsdownsample):** Provides optimized implementations of downsampling algorithms that help to maintain important aspects of the data.\n",
+    "\n",
+    "### Considerations and Trade-offs\n",
+    "While this approach allows visualization and interaction with datasets larger than available memory, it does introduce certain trade-offs:\n",
+    "\n",
+    "- **Increased Storage Requirement:** Constructing a data pyramid currently requires additional disk space since multiple representations of the data are stored.\n",
+    "- **Code Complexity:** Creating the pyramids still involves a fair bit of familiarity with the key packages, and their interoperability. Also, the plotting code involved in dynamic access to the data pyramid structure is still experimental, and could be matured into HoloViz or another codebase in the future.\n",
+    "- **Performance:** While this method can handle large datasets, the performance may not match that of handling smaller datasets due to the overhead associated with processing and dynamically loading multiple layers of the pyramid."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports and Configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "import numpy as np\n",
+    "import h5py\n",
+    "import dask.array as da\n",
+    "import xarray as xr\n",
+    "from xarray.backends.api import open_datatree # until API stabilizes\n",
+    "\n",
+    "import panel as pn\n",
+    "import holoviews as hv\n",
+    "\n",
+    "from ndpyramid import pyramid_create\n",
+    "from tsdownsample import MinMaxLTTBDownsampler\n",
+    "\n",
+    "from IPython.display import display\n",
+    "\n",
+    "hv.extension(\"bokeh\")\n",
+    "pn.extension()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_URL = 'https://datasets.holoviz.org/lfp/v1/sub-719828686_ses-754312389_probe-756781563_ecephys.nwb'\n",
```

Comment:
I respectfully disagree. This particular dataset is 900MB and we need to warn users before they download it. I think it's necessary to be very explicit in this case.

Reply:
But we have so many examples that use this pattern, and some of them download bigger files than that, for example `census` with a 1.3 GB file. On top of that, building a conda environment (from an empty cache) often requires downloading even more data than that. So, while I'd concede there could be a better UX, letting people accept whether they want to execute these expensive internet downloads or not, this has so far been the adopted pattern across all the examples.
Using `anaconda-project` to download also allows us to set up `test_data`, which is not possible (well, not implemented), if you inline the downloads in the notebook.

I'm sure we could improve things, but let's start with opening an issue, discussing, and then implementing changes across all the examples, instead of special casing some of them.

Reply:
added data listing to anaconda-project